src/Model/List/Map.php

```diff
@@ -74,7 +74,8 @@ public function toArray()
         $array = [];
 
         foreach ($this as $k => $v) {
-            $array[$k] = $v;
+            $key = $k ?? '';
+            $array[$key] = $v;
```

Comment:
```suggestion
            $array[$k ?? ''] = $v;
```
This would be tidier - but I won't block merging if this change isn't made.

Reply:
Will leave as is, I prefer it longer form